CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ It is based on Tidepool Blip 1.27.
 ### Changed
 - YLP-2574 Create tool tips independently of the name of the columns
 - YLP-2749 Use a branded Verify Email result page instead of the Auth0 one
+- YLP-2122 Remove JP and GB from the list of countries
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
**🤓 nitpick:** ‏Tickets are ordered based on their number, if this one could be ordered in the list it would be great ✨